### DISCUSSION TITLE
How to get to DAX Guide from the editing window

### DISCUSSION THREAD

**Discussion by erapade** [DaxStudio/DaxStudio#567] [Q&A] [answered]
When writing code, it's obvious how to get to the DAX-guide, you just write the function name and then you get a menu with a link  to the function in the DAX guide.

But what if yu already have all the code written and you would like to look-up a function in DAX guide. I can't find any right-click option for this, have I missed something?

![image](https://user-images.githubusercontent.com/17324892/110938227-d8df0f80-8333-11eb-930c-3081a44819a0.png)

And by the way... In DAX guide I miss the links to MS documentation that was there before. Yes, sometimes they have information missing in the DAX-guide

**Comment by dgosbell** [comment, marked as answer]
At the moment the function tips are only triggered when you type a '(' character. So the only work around at present is to delete then re-type the '(' character (not great, but it works)

>  I can't find any right-click option for this, have I missed something?

No you have not missed anything. A right-click option in the editor is not something I'd though of before and no one else has suggested it before this. I think it would be helpful and not too hard to implement. I've created a feature request at #568 to track this idea.

> And by the way... In DAX guide I miss the links to MS documentation that was there before. Yes, sometimes they have information missing in the DAX-guide

dax.guide is a separate website not part of DAX Studio however I can still see the links to the ms documentation so this may have been a temporary thing.

**Comment by marcosqlbi** [comment]
The link to MSDN documentation is available in DAX Guide: just check for "documentation" at the bottom of each function (for those functions that are documented - there are several functions not available on Microsoft documentation).
